src/index.js

```diff
@@ -2,7 +2,8 @@
  * Copyright 2015, Yahoo! Inc.
  * Copyrights licensed under the New BSD License. See the accompanying LICENSE file for terms.
  */
-const ReactIs = require('react-is');
+import * as ReactIs from 'react-is';
```

Comment:
this was a change I was after, but it required some config changes to make the whole thing work as expected (UMD builds were broken)

Reply:
If we use the wildcard here do we still need to specify `namedExports` in rollup? Might as well import only what's required. Make's it ready for a future react-is esm build and tree shaking in webpack.

I suspect if anything only rollup will be able to tree-shake unrelated exports from `react-is` currently. Webpack would need an esm build for `react-is`, correct?

Reply:
I think in this particular case even rollup is not quite able to tree-shake react-is because of the dev/prod conditional exports in it. I haven't actually tested this though. 

Similarly because of this dev/prod conditional export we have to currently specify `namedExports` because otherwise rollup is not able to statically analyze the shape of `ReactIs` and it warns us with:
```
(!) Missing exports
https://rollupjs.org/guide/en#error-name-is-not-exported-by-module-
src/index.js
ForwardRef is not exported by node_modules/react-is/index.js
```